### DISCUSSION TITLE
destination/credential datasources removed from tf provider

### DISCUSSION THREAD

**Discussion by log1cb0mb** [infrahq/infra#3883] [General]
With reference to issue reported here:
https://github.com/infrahq/terraform-provider-infra/issues/16

Why? :(

**Comment by mxyng** [comment]
Summary from the linked issue:

The `infra_destination` and `infra_credential` data sources were removed because there wasn't a strong  use case at the time. At the same time, the data source is limited by the credentials API resource which can lead to unexpected errors when created as a Terraform data source. Once we've sorted out this issue, we'll re-evaluate the fate of these data sources.
